### DISCUSSION TITLE
希望可以有检查更新的功能

### DISCUSSION THREAD

**Discussion by yesanshao007** [hiroi-sora/Umi-OCR#329] [Ideas]
软件肯定会不断地迭代，但是目前想要体验新软件，就得上github获取最新软件，这样有些不便，希望可以有检查更新的功能，有新版本可以下载

**Comment by hiroi-sora** [comment]
了解。检查更新是很有必要的功能。目前只是我忙且懒所以一直没整😂，在后续版本肯定会出的。
